consumer.go

```diff
@@ -617,6 +626,12 @@ func (child *partitionConsumer) parseResponse(response *FetchResponse) ([]*Consu
 	child.fetchSize = child.conf.Consumer.Fetch.Default
 	atomic.StoreInt64(&child.highWaterMarkOffset, block.HighWaterMarkOffset)
 
+	if response.Version == 11 && len(child.consumer.conf.RackID) > 0 {
```

Comment:
I accidently removed my suggestion. Doing it again.

To support additional Fetch protocol changes, the comparison with version 11 should allow this.  
change
   **response.Version == 11**
to 
   **response.Version >= 11**